src/templates/parts/screen-cell.hbs

```diff
@@ -7,6 +7,7 @@
     {{#if this.spanCols}}grid-column-end: span {{this.spanCols}};{{/if}}
       --column-span-count: {{#if this.spanCols}}{{this.spanCols}}{{else}}1{{/if}}">
 
+  {{#if @root/isGM}}
```

Comment:
Instead of fully preventing this element from rendering it should render the Title of the entity displayed (instead of a select), and should show the "Open" button.

Reply:
Agreed, will look into it today

Reply:
Done! Updated screenshot up there. You can see players now have a button to open journal.